### PR TITLE
feat: add group feature flag details

### PR DESCRIPTION
**Pull request by liyiy**
## Changes

Screen Shot 2022-09-12 at 4 44 41 PM

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/scenes/groups/Group.tsx [hunk 1]
```diff
@@ -95,6 +96,15 @@ export function Group(): JSX.Element {
                 >
                     <RelatedGroups id={groupKey} groupTypeIndex={groupTypeIndex} />
                 </TabPane>
+                <TabPane
+                    tab={<span data-attr="groups-related-flags-tab">Feature flags</span>}
+                    key={PersonsTabType.FEATURE_FLAGS}
+                >
+                    <RelatedFeatureFlags
+                        distinctId={groupData.group_key}
+                        groups={{ [groupTypeName]: groupDisplayId(groupKey, groupData.group_properties) }}
```
Comment (liyiy) [inline frontend/src/scenes/groups/Group.tsx:105]: @neilkakkar I referred to the tests but do you mind double checking if I'm passing in the groups correctly? 😬

Comment (neilkakkar) [inline frontend/src/scenes/groups/Group.tsx:105]:
```suggestion
                        groups={{ [groupTypeName]: groupKey }}
```

**Review by neilkakkar — APPROVED** [review]
Lol, this was smaller than I expected, nice work!

**Comment by liyiy** [comment]
> Lol, this was smaller than I expected, nice work!

thank god for reuseable, extensible code 😆

**Pull request merged**